content/docs/06-integrations/00-aws-autoscaler.md

```diff
@@ -14,81 +14,174 @@ import InfoBox from 'shared/components/InfoBox';
+<br />
+
+### Define IAM Policy
+Define the new IAM policy, using the full Cluster Autoscaler policy outlined in the prerequisites section above, and give it a name, for example, *PaletteEKSClusterAutoscaler*. After defining the IAM policy, copy its ARN to use in the next step. For example, the policy ARN will be similar to `arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler`. 
+<br />
+
+### Update the Kubernetes Layer Manifest 
+Log in to Palette, and create a new cluster profile for AWS EKS or update an existing one. In the Kubernetes layer of your cluster profile, update the `managedMachinePool.roleAdditionalPolicies` section in the Kubernetes pack's manifest with the newly created IAM policy ARN. The snapshot below shows the section to update with the policy ARN. 
+
+
+![Snapshot showing the ARN  added to the Kubernetes pack's manifest.](/integrations_aws-cluster-autoscaler_k8s-manifest.png)
+
+
+The code block below shows the updated `managedMachinePool.roleAdditionalPolicies` section with a sample policy ARN, `"arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler"`, as an example. Before you use the following code block, replace the ARN below with yours. 
+<br />
+
+  ```yaml
+  managedMachinePool:
+    # roleName: {{ name of the self-managed role | format "${string}" }}
+    # A list of additional policies to attach to the node group role
+    roleAdditionalPolicies:
+    - "arn:aws:iam::650628870702:policy/PaletteEKSClusterAutoscaler"
+  ```
+  <br />
+
+<InfoBox>
+
+If you do not want to update the Kubernetes pack's manifest, you can add an *inline* IAM policy to the cluster's node group post deployment. Refer to this [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console) on how to embed an inline policy for a user or role.
+
+</InfoBox>
+
+
+Note that when you deploy an EKS cluster using Palette, Palette creates two IAM roles in the AWS account automatically - one for the cluster and another for the node group. Adding the ARN of your newly created IAM policy to the Kubernetes pack's manifest will make Palette add that IAM policy to the node group during deployment. 
+<br />
+
+### Add Cluster Autoscaler Pack
+After updating your Kubernetes pack, add/update other layers per your requirements. Next, add AWS Cluster Autoscaler pack as an add-on layer. 
+<br />
+
+### Deploy a Cluster
+Once your cluster profile is complete, deploy it to a cluster. During the cluster configuration, ensure that your worker pool is made up of instance type **t3.2xlarge** (8 vCPUs, 32 GB RAM) or higher. As an example, the snapshot below shows one node of type **t3.2xlarge** in the worker pool of a successfully deployed cluster.  

```

Comment:
This doesn't seem right. The reason I say this is because the autoscaler pack uses the [EC2 Cluster Autoscaler](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md) behind the scenes and that has no minimum reqs. What's more important is to convey to the user that they should ensure each instance size has adequate resources that provide approximately equal capacity. 

![CleanShot 2023-07-13 at 11 39 22](https://github.com/spectrocloud/librarium/assets/29551334/848f922a-7826-435a-a733-918dd88a7a9e)